2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,38 @@
   You should write and log at least 5 recipes
 */
 
```

Comment:
The recipe objects look good 👍 
But take another look at the instructions on lines 13-20. It looks each recipe should be output in this way:
```
  Mole
  Serves: 2
  Ingredients:
  cinnamon
  cumin
  cocoa
```